main/consolekit/pam-foreground-compat.ck

```diff
@@ -1,17 +0,0 @@
-#!/bin/sh
```

Comment:
Are you 100% sure that this is no longer needed? see commit 85314dd187611074fc409c6cee98720751e499f6